Makefile

```diff
@@ -176,6 +176,8 @@ test/batch: run-cmake-release
 	./build/bin/foedag --batch --script tests/TestBatch/test_compiler_mt.tcl
 	./build/bin/foedag --batch --script tests/TestBatch/test_compiler_stop.tcl
 	./build/bin/foedag --batch --script tests/TestBatch/test_compiler_batch.tcl
+	./build/bin/foedag --batch --script tests/TestBatch/task_clean.tcl
```

Comment:
Does the test_task_clean.tcl name align with other test filenames?

Reply:
Renamed to test_task_clean.tcl